azure-pipelines.yml

```diff
@@ -263,6 +269,7 @@ stages:
           MIN_WORKER: $(T0_SONIC_INSTANCE_NUM)
           MAX_WORKER: $(T0_SONIC_INSTANCE_NUM)
           MGMT_BRANCH: $(BUILD_BRANCH)
+          PTF_IMAGE_TAG: $(BUILD_BRANCH)
```

Comment:
@opcoder0 Do we need additional change in sonic-mgmt to use branch tag?

Reply:
@kperumalbfn all the branches (master and release) seem to use the latest tag by default. For instance as configured in https://github.com/sonic-net/sonic-mgmt/blob/202405/setup-container.sh#L141C8-L141C87

Reply:
@kperumalbfn the pipelines in buildimage and sonic-mgmt repo all use the pipeline on master branch which accepts parameter to override PTF_IMAGE_TAG. Each release branch now overrides the tag. It wasn't required before as all branches used mixed image. Now master uses py3 only image and hence the override.

Reply:
@opcoder0 could you update this in master and that will be used for future releases as well.